doc/nrf/ug_matter_hw_requirements.rst

```diff
@@ -162,3 +166,293 @@ Values are provided in kilobytes (KB).
+        +-----------------------------------------+----------------+-------------------+---------------------+-----------------+-----------------+
+
+      Internal flash (0x100000 = 1024kB)
+        +-----------------------------------------+---------------------+-------------------+---------------------+-----------------+-----------------+
+        | Partition                               | Offset              | Size              | Partition elements  | Element offset  | Element size    |
+        +=========================================+=====================+===================+=====================+=================+=================+
+        | Bootloader (mcuboot)                    | 0kB (0x0)           | 28kB (0x7000)     | -                   | -               | -               |
+        +-----------------------------------------+---------------------+-------------------+---------------------+-----------------+-----------------+
+        | Application (mcuboot_primary)           | 28kB (0x7000)       | 976kB (0xf4000)   | mcuboot_pad         | 28kB (0x7000)   | 512B (0x200)    |
+        |                                         |                     |                   +---------------------+-----------------+-----------------+
+        |                                         |                     |                   | mcuboot_primary_app | 28.5kB (0x7200) | 975kB (0xf3e00) |
+        +-----------------------------------------+---------------------+-------------------+---------------------+-----------------+-----------------+
+        | Factory data (factory_data)             | 1004kB (0xfb000)    | 4kB (0x1000)      | -                   | -               | -               |
+        +-----------------------------------------+---------------------+-------------------+---------------------+-----------------+-----------------+
+        | Non-volatile storage (settings_storage) | 1008kB (0xfc000)    | 16kB (0x4000)     | -                   | -               | -               |
+        +--------------------------+---------------------+-------------------+---------------------+-----------------+-----------------+
+
+      SRAM primary (0x40000 = 256kB)
+        +-----------------------------------------+----------------------+-------------------+---------------------+-----------------+-----------------+
+        | Partition                               | Offset               | Size              | Partition elements  | Element offset  | Element size    |
+        +=========================================+======================+===================+=====================+=================+=================+
+        | Static RAM (sram_primary)               | SRAM is located at the address ``0x20000000`` in the memory address space of the application.   | 256kB (0x40000)   | -   | -   | - |
```

Comment:
Shouldn't the comment be placed before the table? It's not about "offset" but **address** of the memory region.